app/src/main/java/bankingapp/App.java

```diff
@@ -28,8 +37,8 @@ public static void main(String[] args) {
             try (Statement statement = connection.createStatement()) {
 
                 //Create DB
-                DBTrans db = new DBTrans();
-                db.createDB(statement);
+                DBTransactions db = new DBTransactions();
```

Comment:
```suggestion
                IDBTransactions db = new DBTransactions();
```

Use the interface here. That way, you can be sure that everything main needs to operate is included in the interface.